uint8 leaf_id
uint16 addr
uint16 count
---

uint8 exception_code
uint8 len
uint16[] values